modules/apps/dynamic-data-mapping/dynamic-data-mapping-form-web/src/main/java/com/liferay/dynamic/data/mapping/form/web/internal/upgrade/v1_1_0/UpgradeDDMFormPortletPreferences.java

```diff
@@ -0,0 +1,77 @@
+/**
+ * SPDX-FileCopyrightText: (c) 2024 Liferay, Inc. https://liferay.com
+ * SPDX-License-Identifier: LGPL-2.1-or-later OR LicenseRef-Liferay-DXP-EULA-2.0.0-2023-06
+ */
+
+package com.liferay.dynamic.data.mapping.form.web.internal.upgrade.v1_1_0;
+
+import com.liferay.dynamic.data.mapping.constants.DDMPortletKeys;
+import com.liferay.petra.string.StringBundler;
+import com.liferay.petra.string.StringPool;
+import com.liferay.portal.kernel.portlet.PortletPreferencesFactoryUtil;
+import com.liferay.portal.kernel.upgrade.BasePortletPreferencesUpgradeProcess;
+import com.liferay.portal.kernel.util.GetterUtil;
+import com.liferay.portal.kernel.util.Validator;
+
+import java.sql.PreparedStatement;
+import java.sql.ResultSet;
+
+import javax.portlet.PortletPreferences;
+
+/**
+ * @author Paulo Albuquerque
+ */
+public class UpgradeDDMFormPortletPreferences
+	extends BasePortletPreferencesUpgradeProcess {
```

Comment:
@p-albuquerque you should always follow the same pattern from the base class. Please rename it to `DDMFormPortletPreferencesUpgradeProcess`.

Reply:
No